src/en/space-station-14/departments/science/proposals/anomalous-research-update.md

```diff
@@ -0,0 +1,31 @@
+# Proposed changes to anomalous research 
+ 
+ | Designers     | Implemented | GitHub Links |
+ |---------------|-------------|--------------|
+ | chromiumboy   | :x: No      | TBD          |
+ 
+ ## Problem statement
+ Currently, the study of station anomalies is somewhat lacklustre. Generally, anomalous research proceeds as follows:
+ -	The science team generates an anomaly with the anomaly generator located in science and then sets out to locate it.
+ -	If the anomaly is not found within a short period, it will likely cause substantial damage to the station and/or crew, either by going critical or via its pulse effects.
+ -	When the anomaly is located, but it is not in a place that is deemed appropriate or it has an undesirable pulse effect, it is generally removed.
+ -	Once an appropriate anomaly has been found and stabilized so that research on it can commence, it is then generally left to its own devices by the science team until it begins losing stability. Unless the pulse effect is easily countered (e.g., spacing a room to prevent the heat effect of a fire anomaly), the anomaly will continue to damage the station when it pulses, much to the annoyance of station engineers.
+ To summarize the frustrations of this gameplay loop, in the initial stages of anomalous research, the science team is sent on what is essentially a wild goose chase in order to find the anomaly in a timely manner. They will likely need to repeat the search until they achieve what is considered an ‘acceptable’ combination of anomaly type and station location. Even then, conducting/maintaining any anomalous research poses a significant ongoing risk to the crew, with little to benefit them.
+ 
+ ## Proposed changes
+ To alleviate the problems listed above, the following changes to the anomalous research gameplay loop are proposed:
+ -	The science team surveys the station using hand portable tools (i.e., repurposed anomaly detectors) to gather data on potential anomaly sites (i.e., space-time rifts. These rifts are invisible to most players, with some notable exceptions, such as the heretic).
```

Comment:
i actually really like this idea. Being able to survey anomaly locations give scientists something to do and encourages interesting record keeping. It also helps speed up searching for anomalies, since people can approximate the location using the pools.